### PR TITLE
claimant representative notification email

### PR DESCRIPTION
**Pull request by btensay**
Notifications for Claimant's Rep on MyHMCTS

### Jira link (if applicable)
https://tools.hmcts.net/jira/browse/RET-5309

### Change description ###
Added Notifications for Claimant's Rep on MyHMCTS

### Checklist

- [ x ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ x ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/SendNotificationService.java [hunk 1]
```diff
@@ -216,13 +221,30 @@ public void sendNotifyEmails(CaseDetails caseDetails) {
...
+                    emailService.sendEmail(claimantRepSendNotificationTemplateId, personalisation.get(EMAIL_ADDRESS),
+                            personalisation);
+                }
+            } else {
+                // If not represented, send notification to the claimant Only
+                String claimantEmailAddress = caseData.getClaimantType().getClaimantEmailAddress();
+                if (!isNullOrEmpty(claimantEmailAddress)) {
+                    emailService.sendEmail(claimantSendNotificationTemplateId, claimantEmailAddress,
+                            buildPersonalisation(caseDetails, emailService.getCitizenCaseLink(caseId)));
+                }
```
Comment (hsjhita) [inline src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/SendNotificationService.java:243]: I think should maybe be wrapped in an else if instead where we check to see if the claimant is a system user as well. There are claims on prod where claimants do not have access to citizen hub so we need to make sure we aren't sending them emails to something they can't access

Comment (btensay) [reply src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/SendNotificationService.java:243]: I add fix to cover that scenario.

**Review by hsjhita — DISMISSED** [review]

**Comment by hsjhita** [comment]
**Note:** Before merging, remember to create the template in the prod gov notify and add to cnp fluc config for et-cos prod.yaml

**Comment by hsjhita** [comment]
Replaced by another branch

**Pull request closed without merging**